cfme/tests/containers/test_manageiq_ansible_providers.py

```diff
@@ -69,10 +93,11 @@ def test_manageiq_ansible_add_provider_same_name(ansible_providers, provider):
         2. Test navigates to Containers Providers page and verifies the provider was not added
 
         """
-    setup_ansible_script(provider, script_type='providers', script='add_provider')
-    run_ansible('add_provider')
-    run_ansible('add_provider')
-    assert get_yml_value('add_provider', 'name') in provider.summary.properties.name.value
+    script_name = 'add_provider'
+    setup_ansible_script(provider, script_type='providers', script=script_name)
+    run_ansible(script_name)
+    run_ansible(script_name)
```

Comment:
Why is it required twice?

Reply:
This test checks adding adding Containers Provider with the same name